src/Avalonia.Controls.TreeDataGrid/TreeDataGridRowEventArgs.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Avalonia.Controls
+{
+    public class TreeDataGridRowEventArgs
+    {
+        public TreeDataGridRowEventArgs(IControl row, int rowIndex)
+        {
+            Row = row;
+            RowIndex = rowIndex;
+        }
+
+        internal TreeDataGridRowEventArgs()
+        {
+            Row = null!;
+        }
+
+        public IControl Row { get; private set; }
```

Comment:
Why IControl though?

Reply:
It was for symmetry with ``TreeDataGridCellEvents` which exposes a `Control` as a cell can technically be any type of control.  Changed to be `TreeDataGridRow` in this case as it can only be a `TreeDataGridRow` here.